### ISSUE TITLE
Erratic PredAI load predictions

### ISSUE THREAD

**Issue by pbyrne-ms** [springfall2008/predai#7]
Erratic PredAI load predictions

PredAI Version: 1.1.7
PredBAT Version: 7.17.2
HA Version: 2024.4.3

For about a week now, my PredAI predicted load entity 'sensor.givtcp1_chxxxxxxxx_load_energy_today_kwh_prediction' has been forecasting extremely large load values. And then, even more erratically, goes back to normal after a few hours.
By Erratic, usually the prediction adds 800kWh-ish to the load..
 
 
YAML file is the most simple example possible

```
update_every: 30
sensors:
  - name: sensor.givtcp_chxxxxxx_load_energy_today_kwh
    #subtract: sensor.car_charger_energy_today
    days: 14
    incrementing: True
    reset_daily: True
    interval: 30
    units: kWh
    future_periods: 96
    #database: True
```

Screenshots:
![image](https://github.com/springfall2008/predai/assets/33572035/8bd1fdb5-0843-4533-bcce-69c8d6ffac53)

Any suggestions? or spot anything wrong?

**Comment by mpartington** [comment]
Think I'm seeing same thing 
<img width="274" alt="image" src="https://github.com/springfall2008/predai/assets/106986397/06f76caa-8b9b-4bc8-b7da-f1526bbf0ccd">

**Comment by stevensoave** [comment]
Seeing similar:

update_every: 5

sensors:
  - name: sensor.givtcp_sd2145g026_load_energy_today_kwh
    subtract: sensor.mixergy_electric_heat_energy
    days: 30
    incrementing: True
    reset_daily: True
    interval: 5
    units: kWh
    future_periods: 96
    database: True
    export_days: 14
    
    
![2024-05-12 16_31_24-Overview – Home Assistant and 174 more pages - Personal - Microsoft​ Edge](https://github.com/springfall2008/predai/assets/24212589/5333eb2a-9fab-46bf-b48e-6c628e910289)

**Comment by pbyrne-ms** [comment]
closing issue. did not track exactly which update resolved it, but as of 28-May-2024 and PredAI 1.1.13 the issue has been resolved. Predictions are now stable. Also deleted any local .db files, and they get auto re-created, so its a good way to flush out any crud accumulated within the files.
